### ISSUE TITLE
spelling is incorrect

### ISSUE THREAD

**Issue by rijndaelsy** [SamsungARTIK/artik-sdk-examples#1]
artik_error(*get_celcius) (artik_sensor_handle handle,
					   int *store);
		/*!
		 *  \brief get_fahrenheit permit to return the degree of environement in fahrenheit
		 *
		 *  \param[in] handle handle tied to the requested TEMPERATURE communication module instance to be released.
		 *                  This handle is returned by the 'get_temperature_sensor' function.
		 *  \param[in] store permit to save the result of the function.
		 *
		 *  \return S_OK and store the value into the parameter on success, error code otherwise
		 */
		artik_error(*get_fahrenheit) (artik_sensor_handle handle,
					      int *store);

	} artik_sensor_temperature;
in the library and in various examples
celsius is mentioned creating error

**Comment by glemercier** [comment]
@rijndaelsy, thanks for the report. We fixed it in ARTIK SDK 1.3.0 which just got released, the examples on this repo will be updated shortly to match the API changes in this new version.

**Comment by rijndaelsy** [comment]
@glemercier  sir there are other issues also.Can you please tell me when is ARTIK SDK 1.3.0 releasing.As i am using artik board in my internship i can surely test and bring out issuees

**Comment by glemercier** [comment]
@rijndaelsy, the examples have been updated and now match ARTIK SDK 1.3.0. To update your ARTIK board to this new version, please refer to this page: https://developer.artik.io/documentation/api/ARTIK.html => short answer: "dnf makecache --refresh && dnf update".
Thanks for using the ARTIK SDK for your internship needs, feel free to report any bugs or issues.

**Comment by rijndaelsy** [comment]
sir,
as we are updating through the link, even then also libraries are not updated on the artik.i have followed every method in the documentation please suggest some way as I am stuck in the above only

**Comment by glemercier** [comment]
What does "dnf info libartik-sdk-base" say on your board?

**Comment by rijndaelsy** [comment]
dnf info libartik-sdk-base
Last metadata expiration check performed 3:25:21 ago on Tue Jun 20 01:53:02 2017.
Installed Packages
Name        : libartik-sdk-base
Arch        : armv7hl
Epoch       : 0
Version     : 1.0
Release     : 2
Size        : 35 k
Repo        : @System
From repo   : artik
Summary     : Base package needed by all the other packages of the SDK
URL         : http://www.artik.io
License     : Proprietary
Description : This library contains the base library of the SDK for addressing
            : common features


cat /etc/artik_release
RELEASE_VERSION=1020GC0F-3AF-01Q2
RELEASE_DATE=20160418.203719
RELEASE_UBOOT=U-Boot 2012.07
RELEASE_KERNEL=3.10.93

**Comment by rijndaelsy** [comment]
even the node.js examples are giving error about libartik-sdk-mqtt library.sir i am very confused and dwindling from different architecture to different languages and api.I am stuck.

**Comment by glemercier** [comment]
Ok I see. Unfortunately you are using an ARTIK 1020 board that is no longer supported by Samsung. Therefore the last system image you can get is running Fedora 22, for which we dropped support like a year ago. We only release latest versions of the SDK to the ARTIK boards running Fedora 24, this is the reason why you are not getting version 1.3.0 when you do "dnf update".

You should still be able to run the examples if you checkout commit 65fc97a21c2d01d572e9ad3f49022e10dc94119f. Of course you'll be limited to the APIs supported in 1.0.2. For using more recent versions of the ARTIK SDK, I have no other solution than recommending you to purchase an ARTIK530 or ARTIK710 :-(

**Comment by rijndaelsy** [comment]
sir is artik 520 supported or not

**Comment by rijndaelsy** [comment]
https://www.artik.io/blog/2017/02/recent-updates-for-developers/
in the above link it was mentioned that fedora 24 available for artik 520 so i want to know whether it is supported or not

**Comment by glemercier** [comment]
Yes, A520 would work. You can download latest 2.0.0 image from here: http://developer.artik.io/downloads/

**Comment by rijndaelsy** [comment]
Sir as i am lagging very much in my internship i wanna ask you about the capabilities of those apis or we need some other stack on the top of it to make a smart grid system
Thank you

**Comment by glemercier** [comment]
It depends on what your smart grid system contains :-) The ARTIK SDK will not provide you with a full stack to develop such a specific thing, it will give you APIs to address remote devices or chips connected to the ARTIK board in an easy way. It's basically a set of APIs to ease up development using most common IoT protocols and hardware busses.

The actual application code and business logic (in your case the smart grid system) remains to be developed for each one's own use case.
